### PR TITLE
Update documentation for v1.9.2

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

**Review by Falke-Design — APPROVED** [review]

File: docs/download.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ bodyclass: download-page
 		<th>Description</th>
 	</tr>
 	<tr>
-		<td><a href="https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/v1.9.1/leaflet.zip">Leaflet 1.9.1</a></td>
-		<td>Stable version, released on September 23, 2022.</td>
+		<td><a href="https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/v1.9.2/leaflet.zip">Leaflet 1.9.2</a></td>
+		<td>Stable version, released on October 4, 2022.</td>
 	</tr>
 	<tr>
 		<td><a href="https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/v1.8.0/leaflet.zip">Leaflet 1.8.0</a></td>
```
Comment (jonkoops) [inline docs/download.md:19]: This still lists 1.8.0, not 1.9.1. Perhaps this was intentional? @mourner

**Comment by Falke-Design** [comment]
Do we want to change the banner?
grafik

File: docs/download.md (hunk 1 above)
Comment (mourner) [reply docs/download.md:19]: Yeah, let's keep it for now.

**Comment by mourner** [comment]
> Do we want to change the banner?

No, I think it's good. I added notes about v1.9.1/v1.9.2 in the full changelog the post links to.

**Comment by Falke-Design** [comment]
But then let us add a info for dropping ESM

grafik

**Review by Falke-Design — APPROVED** [review]

**Comment by jonkoops** [comment]
@mourner this good to go in your book?

**Review by mourner — APPROVED** [review]

**Comment by jonkoops** [comment]
Cherry-picked this commit on `v1` as well.

**Pull request merged**